source/docs/assist/voice_remote_local_assistant.markdown

```diff
@@ -46,4 +46,6 @@ For the quickest way to get your local Assist pipeline started, follow these ste
 You would like to tweak the configuration for better performance or accuracy?
 View some of the options in the video below. Explained by Mike Hansen, creator of Rhasspy, Piper, and Wyoming.
 
-<lite-youtube videoid="Tk-pnm7FY7c" videoStartAt="1589" videotitle="Configure your local Assist pipeline for your setup"></lite-youtube>
\ No newline at end of file
+<lite-youtube videoid="Tk-pnm7FY7c" videoStartAt="1589" videotitle="Configure your local Assist pipeline for your setup"></lite-youtube>
+
+The options are also documented in the add-on itself. Go to the {% my supervisor_addon addon="core_whisper" title="**Whisper**" %} or the {% my supervisor_addon addon="core_piper" title="**Piper**" %} add-on and open the **Documentation** page.
```

Comment:
Adding a new line, not because it is really needed, but because something is from @ Netlify and trying to trigger a rebuild.

```suggestion
The options are also documented in the add-on itself. Go to the {% my supervisor_addon addon="core_whisper" title="**Whisper**" %} or the {% my supervisor_addon addon="core_piper" title="**Piper**" %} add-on and open the **Documentation** page.

```